assets/sass/_general.scss

```diff
@@ -6,6 +6,8 @@ html,
 body {
   font-size: $fontSize;
   background-color: $background;
+  //TODO investigate which element causes horizontal scroll
+  overflow-x: hidden;
```

Comment:
Please fix this. Before the changes there was no overflow. Is the overflow because of the padding?

Reply:
`width: 100vw;` was causing this problem.
`max-width: 100%;` fixed this problem